<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Math Parser</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>629</width>
      <height>350</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="0">
      <widget class="QTextBrowser" name="error_box"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Error info:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <property name="horizontalSpacing">
        <number>15</number>
       </property>
       <property name="verticalSpacing">
        <number>7</number>
       </property>
       <item row="7" column="3" rowspan="2">
        <widget class="QRadioButton" name="Radians_Button">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Radians</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="9" column="3">
        <widget class="QRadioButton" name="Degrees_Button">
         <property name="text">
          <string>Degrees</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QLineEdit" name="infix_box"/>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QTextBrowser" name="rpn_box"/>
       </item>
       <item row="4" column="2" rowspan="6">
        <widget class="QTextBrowser" name="result_box">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1" rowspan="6">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>   Result:    </string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>RPN representation:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Enter an expression to calculate:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3" rowspan="5">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>List of functions and operators you can use:
#  +, -, *, /, decimal dot: .
#  sin(x), cos(x), tan(x), ctg(x)
#  pow(x) (exponent = 2), sqr(x), exp(x)
Other characters and functions forbidded.
Note: functions cannot take expressions
as an argument, single numbers only
</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" rowspan="6">
        <widget class="QPushButton" name="run">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Calculate value</string>
         </property>
        </widget>
       </item>
       <item row="6" column="3">
        <widget class="QRadioButton" name="Recursive_Button">
         <property name="text">
          <string>Recursive Algorithm</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_2</string>
         </attribute>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QRadioButton" name="Djikstra_Button">
         <property name="text">
          <string>Djikstra's Algorithm</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_2</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
